CONTRIBUTING.md

```diff
@@ -1,41 +1,82 @@
 # Contributing to OSEM
-Our open source project is open to contributions. Open your [issue](https://github.com/openSUSE/osem/issues/new) and send a PR!
+We here at OSEM are open for all types of contributions from anyone. Tell us about our [issues/ideas](https://github.com/openSUSE/osem/issues/new), propose code changes via [pull requests](https://help.github.com/articles/using-pull-requests) or contribute artwork and documentation.
 
-We welcome new developers, and we actively participate in mentoring programs, such as [GSoC](https://summerofcode.withgoogle.com/) and [RGSoC](https://railsgirlssummerofcode.org/).
+We welcome all new developers and are also prepared to mentor you through your first contributions! All OSEM maintainers are seasoned developers and have participated in mentoring programs, such as [GSoC](https://summerofcode.withgoogle.com/) and [RGSoC](https://railsgirlssummerofcode.org/).
 
-## Request for contributions
-We are always looking for contributions to OSEM. Read this guide on how to do that.
+We need your input and contributions to OSEM. In particular we seek the following types:
 
-In particular, this community seeks the following types of contributions:
+* **code**: contribute your expertise in an area by helping us expand OSEM with features/bugfixes/UX
+* **code editing**: fix typos, clarify language, and generally improve the quality of the content of OSEM
+* **ideas**: participate in an issues thread or start your own to have your voice heard
 
-* code: contribute your expertise in an area by helping us expand OSEM
-* ideas: participate in an issues thread or start your own to have your voice heard.
-* code editing: fix typos, clarify language, and generally improve the quality of the content of OSEM
+Read this guide on how to do that.
 
-## Running OSEM in development
-We are using [Vagrant](https://www.vagrantup.com/) to create our development environments.
+## How to contribute code
+1. Fork the repository and make a pull-request with your changes
+    1. Make sure that the test suite passes that you comply to our code style.
```

Comment:
I think there is an _and_ missed. Although code style is part of the test suite. :confused: